Quadriga/src/main/webapp/WEB-INF/views/editing/networkvisual.jsp

```diff
@@ -52,4 +52,90 @@ $("input[type=button]").button().click(function(event) {
 
 
 <div id="log"></div>
+<script type="text/javascript">
+
+  function goFullscreen(id) {
+	  document.getElementById('chart').style='position: fixed; top: 0; bottom :0; left: 0; right :0';
+    // Get the element that we want to take into fullscreen mode
+    var a = document.getElementById('chart');
+    document.getElementsByTagName('svg')[0].id = 'svg_id';
+    var svgDoc= document.getElementById('svg_id');

```

Comment:
why do you first get the element by its tag name, set the id, and then get it by its id? Why don't you just get it by its tagname right away?